src/extensions/verify.py

```diff
@@ -38,12 +38,16 @@ async def verify_command(
         reason="Verified Redbrick user.",
     )
 
+    role = plugin.client.find_command(hikari.CommandType.SLASH, "role")
+    assert isinstance(role, arc.SlashCommand)
```

Comment:
I think you want to find `"role add"` and then it should be `SlashSubCommand`, but you'll have to check